src/platforms/node/common/integrations/default-integrations.mdx

```diff
@@ -62,27 +78,27 @@ Available options:
 
 _Import name: `Sentry.Integrations.OnUnhandledRejection`_
 
-This integration attaches global unhandled rejection handlers. By default, all unhandled rejections trigger a warning and log the error. You can change this behavior using the `mode` option, which is in line with Node's CLI options: https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode
+This integration attaches a global unhandled rejection handler. By default, all unhandled rejections trigger a warning and log the error. You can change this behavior using the `mode` option, which is in line with Node's CLI options: https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode.
 
 Available options:
 
 ```js
 {
-  mode: "none" | "warn" | "strict";
+  mode: "none" | "warn" | "strict"; // default: "warn"
 }
 ```
 
 ### LinkedErrors
 
 _Import name: `Sentry.Integrations.LinkedErrors`_
 
-This integration allows you to configure linked errors. They'll be recursively read up to a specified limit and lookup will be performed by a specific key. By default, the limit sets to 5 and the key used is `cause`.
+This integration allows you to configure linked errors. They'll be recursively read up to a specified limit and lookup will be performed by a specific key. By default, the limit is set to 5 and the key used is `"cause"`.
 
 Available options:
 
 ```js
 {
-  key: string;
-  limit: number;
+  key: string; // default: "cause"
```

Comment:
could use typescript syntax here

```typescript
{
  key: string = "cause"
```

Reply:
Can you do that in TS? Everything I've read says you can't provide defaults to an interface.

Reply:
technically its:

```typescript
{
  key: string
}: {
  key = "cause"
}
```

but the version i gave i think is readable enough

Reply:
We don't have a standardized syntax for TS/JS options, but if I'd have to choose, I prefer the former version, the one that Katie used above. It' just more "obvious" imo.

Reply:
there is standardized HTML for options that we use everywhere (definition lists) so thats also a path forward. i didnt look at the context of what these are specifically, but its possible they dont belong in a code block to begin with.

Reply:
It is possible. I was just going with the formatting that was already there. I'll let docs folks make that call. In any case, work for a future PR.